src/module.ts

```diff
@@ -109,6 +109,7 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Watch the Tailwind config file to restart the server
     if (nuxt.options.dev) {
+      nuxt.options.watch = nuxt.options.watch ?? []
```

Comment:
I think you can use `|| []` because any falsy value would fail with next line trying to call `.push`

Reply:
Done! :)